sdk/tables/Azure.Data.Tables/samples/Sample1CreateDeleteTables.md

```diff
@@ -15,24 +13,26 @@ var serviceClient = new TableServiceClient(
     new TableSharedKeyCredential(accountName, storageAccountKey));
 ```
 
-## Creating a Table
-To create a table, create a `TableClient` by invoking `CreateTable` from the `TableServiceClient`. A `TableClient` represents a single table and performs table-level operations like inserting and deleting entities within that table. `tableName` can be replaced with any string to be the name of the table. Tables must have unique names.
+## Create a table
+To create a table, invoke `CreateTable` with a [unique table name](https://docs.microsoft.com/en-us/rest/api/storageservices/understanding-the-table-service-data-model#table-names). The `tableName` can be replaced with any string to be the name of the table.
 
 ```C# Snippet:TablesSample1CreateTable
 // Create a new table. The <see cref="TableItem" /> class stores properties of the created table.
 TableItem table = serviceClient.CreateTable(tableName);
 Console.WriteLine($"The created table's name is {table.TableName}.");
 ```
 
-## Deleting a Table
-To delete the table, invoke `DeleteTable` from the service client with the table name.
+A `TableClient` is needed to perform table-level operations like inserting and deleting entities within the table. To get the `TableClient`, invoke `GetTableClient` with the table name.
```

Comment:
it will be good to add a snippet for this call

Reply:
If I'm not doing anything with the `TableClient`, should that still be there?

I think I wasn't sure about where to put creating a `TableClient` because it can also create a table, which might make it belong in the first sample. But since the samples involving entities actually requires a `TableClient`, I wasn't sure if repeating how to create a table would be necessary. Does that make sense?

Reply:
hmm what about adding a comment about, for more information about this operations see sample `Create Delete entities` or something like that